docs/build/configure/triggers.md

```diff
@@ -60,28 +60,42 @@ Be aware that the component must return the type of data you configure in the tr
 {{% /tab %}}
 {{< /tabs >}}
 
-6. Replace the URL value with the URL of your cloud/lambda function.
+6. Replace the **URL** value with the URL of your cloud function or lambda.
 
    ![The trigger configured with an example URL in the Viam app.](/build/configure/trigger-configured.png)
 
 {{% /tab %}}
-{{% tab name="Raw JSON" %}}
+{{% tab name="JSON mode" %}}
 
-If you prefer to configure your trigger with raw JSON instead of the config builder, you can paste one of the following JSON templates into your JSON config.
-`"triggers"` is a top-level section like `"components"` or `"services"`.
+To configure your trigger by using **JSON** mode instead of **Builder** mode, paste one of the following JSON templates into your JSON config.
+`"triggers"` is a top-level section, similar to `"components"` or `"services"`.
 
 {{< tabs >}}
 {{% tab name="JSON Template: Data Synced" %}}
 
 ```json {class="line-numbers linkable-line-numbers"}
   "triggers": [
     {
-      "url": "<Insert your own cloud function or lambda URL for sending the event>",
+      "name": "<trigger name>",
       "event": {
         "type": "part_data_ingested",
-        "attributes": {
+        "data_ingested": {
           "data_types": ["binary", "tabular", "file"]
         }
+      },
+      "notifications": [
+        {
+          "type": "webhook",
+          "value": "<https://1abcde2ab3cd4efg5abcdefgh10zyxwv.lambda-url.us-east-1.on.aws>",
```

Comment:
The url shouldn't be surrounded with `<` and `>` ... just want to make sure this was intentional if you added for a reason